mvc/src/main/java/cz/muni/fi/controllers/DriverController.java

```diff
@@ -0,0 +1,131 @@
+package cz.muni.fi.controllers;
+
+import cz.muni.fi.dto.DriverCreateDTO;
+import cz.muni.fi.dto.DriverDTO;
+import cz.muni.fi.facade.DriverFacade;
+import javax.validation.Valid;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import org.springframework.web.util.UriComponentsBuilder;
+
+/**
+ * @author Lucie Kureckova, 445264
+ */
+@Controller
+@RequestMapping("/driver")
+public class DriverController {
+    
+    final static Logger log = LoggerFactory.getLogger(DriverController.class);
+
+    @Autowired
+    private DriverFacade driverFacade;
+    
+    /**
```

Comment:
I don't think we need javadoc in controllers.. all methods are documented in facades